src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring.mdx

```diff
@@ -130,6 +130,10 @@ Server-side mobile agent configuration provides a mechanism to remotely manage a
 
 The following setting can be currently modified remotely on the **Application** page:
  * Application Not Responding (ANR) error tracking (Android only): Enables tracking and reporting of the reasons behind ANR errors. This functionality is enabled by default and requires Android agent 7.4.1 or higher.
+    <Callout variant="important">
+    Server-side configuration always take precedence over local agent configuration.
+    For mobile apps with <100,000 concurrently active devices, it may take up to 10 minutes to apply changes to end-user devices. For apps with >100,000 active devices, it may take several hours.
```

Comment:
```suggestion
    For mobile apps with less than 100,000 concurrently active devices, it may take up to 10 minutes to apply changes to end-user devices. For apps with greater than 100,000 active devices, it may take several hours.
```